eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -1,280 +1,314 @@
+import itertools
 import json
 import sys
 import urllib.error
 import urllib.parse
 import urllib.request
 import codecs
+from collections import defaultdict
+from types import SimpleNamespace
 
 import jsonschema
 import xlrd
 
-from eva_cttv_pipeline import efo_term, clinvar_record, consequence_type, config
-from eva_cttv_pipeline import evidence_strings as ES
+from eva_cttv_pipeline import efo_term, clinvar_record, consequence_type, config, evidence_strings
 
 
 __author__ = 'Javier Lopez: javild@gmail.com'
 
 
+class Report:
+    def __init__(self, unavailable_efo_dict=None):
```

Comment:
Since this variable is used in line 59 and `len(None)` will raise an exception, the default value should be the empty list.